src/docs/releases/2.1.0.md

```diff
@@ -52,16 +56,22 @@ Previously, the `UsersCanRegister` property controlled which types of registrati
 
 ### Roles Feature
 
+#### Roles Assignment Permissions Consolidated
+
 In the Roles feature, there were previously `AssignRoles` and `AssignRole_{RoleName}` permissions, alongside the Users feature's `AssignRoleToUsers` and `AssignRoleToUsers_{RoleName}` permissions. As these permissions were redundant and the `AssignRoles` permissions were never actually authorized against, `AssignRoles` and `AssignRole_{RoleName}` have been removed in favor of the latter.
 
 !!! warning
     Please review all your recipes and replace occurrences of `AssignRoles` with `AssignRoleToUsers`, and `AssignRole_{RoleName}` with `AssignRoleToUsers_{RoleName}`.
 
-### New **Azure Communication Services SMS** Feature
+### Azure Communication Services SMS Feature
+
+#### New Azure Communication Services SMS Feature
```

Comment:
Also move this section to the bottom of the file after line 77. This way we can highlight new features away from change logs.

```suggestion

## Added Features

### Azure Communication Services SMS Feature

#### New **Azure Communication Services SMS** Feature
```

Reply:
If we have an "Added Features" section, then having at least two levels for each feature is redundant, so I added it without it. See my note about formatting titles above.